src/Relation/Nullary/Reflects.agda

```diff
@@ -8,15 +8,16 @@
 
 module Relation.Nullary.Reflects where
 
+open import Agda.Builtin.Equality
 open import Data.Bool.Base
 open import Data.Empty
 open import Level
-open import Relation.Binary.PropositionalEquality
 open import Relation.Nullary
 
-variable
-  p : Level
-  P : Set p
+private
+  variable
```

Comment:
If this PR cannot make it to v1.2, this fix should at least be
included: we should never declare non-`private` `variable`s.